docs/index.md

```diff
@@ -1291,13 +1291,69 @@ The "HTML" reporter is what you see when running Mocha in the browser.  It looks
 
 [Mochawesome](https://www.npmjs.com/package/mochawesome) is a great alternative to the default HTML reporter.
 
+## Configuring Mocha (Node.js)
```

Comment:
this is a really cool feature, and I'm excited you're leaning on some of yargs' functionality for it.